src/tests/suite/daos_container.c

```diff
@@ -205,6 +205,80 @@ co_attribute(void **state)
 	}
 }
 
+static bool
+ace_has_permissions(struct daos_ace *ace, uint64_t exp_perms)
+{
+	if (ace->dae_access_types != DAOS_ACL_ACCESS_ALLOW) {
+		print_message("Expected access type allow for ACE\n");
+		daos_ace_dump(ace, 0);
+		return false;
+	}
+
+	if (ace->dae_allow_perms != exp_perms) {
```

Comment:
Does this need to match exactly? Could the exp_perms be a subset of what the ace allows?

Reply:
In this test I just wanted to verify what the default ACL was. So I wanted to check the exact values.